app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/integration/IntegrationHandler.java

```diff
@@ -74,11 +75,11 @@
     Creator<Integration>, Deleter<Integration>, Updater<Integration>, Validating<Integration> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IntegrationHandler.class);
-
     final APIGenerator apiGenerator;
     private final EncryptionComponent encryptionSupport;
     private final Inspectors inspectors;
     private final OpenShiftService openShiftService;
+    private final OpenShiftConfigurationProperties openShiftProperties;
```

Comment:
Perhaps we could store only the 3scale management URL here instead of `OpenShiftConfigurationProperties`